common/djangoapps/student/tasks.py

```diff
@@ -48,3 +55,17 @@ def send_activation_email(self, subject, message, from_address, dest_addr):
             exc_info=True
         )
         raise Exception
+
+
+@periodic_task(ignore_result=True, run_every=crontab(hour=settings.FEATURES.get('COURSE_REPORT_HOUR', '*')))
```

Comment:
Where is COURSE_REPORT_HOUR defined, and what is the value set to?

Reply:
FEATURES.COURSE_REPORT_HOUR can be set in server-vars.yml (transformed to lms.env.json) to specify hours to run a job. when not set the default '*' value will be used, which means 'every hour'